stix2elevator/convert_stix.py

```diff
@@ -1841,7 +1888,7 @@ def process_report_contents(report, env, report_instance):
 
 def convert_report(report, env):
     report_instance = create_basic_object("report", report, env)
-    info("Report %s contains only the objects explicitly specified in the STIX 1.x report", 726, report_instance["id"])
+    # info("Report %s contains only the objects explicitly specified in the STIX 1.x report", 726, report_instance["id"])
```

Comment:
Is this message intended to be used still or removed entirely?

Reply:
removed